### PR TITLE
[#1004] Isolate dependencies in repo/db/utxos_test.go

### PR DESCRIPTION
**Pull request by frodeaa**

### CODE REVIEW

**Comment by gubatron** [comment]
can't this be refactored into a function? so much copy/paste.
```
	var uxdb, teardown, err = buildNewUnspentTransactionOutputStore()
	if err != nil {
		t.Fatal(err)
	}
	defer teardown()
	utxo := mustNewUtxo()
	err = uxdb.Put(utxo)
```

**Comment by gubatron** [comment]
sweet! tnx for the refactor

File: repo/db/utxos_test.go [hunk 1]
```diff
@@ -32,11 +26,39 @@ func init() {
 	}
 }
```
Comment (placer14) [inline repo/db/utxos_test.go]: This wants to be a factory that lives in `test/factory/utxo.go`. Example: https://github.com/OpenBazaar/openbazaar-go/blob/master/test/factory/sale_record.go

Comment (placer14) [reply repo/db/utxos_test.go]: Completed in https://github.com/OpenBazaar/openbazaar-go/pull/1265/commits/d2a28fdc497decc195647cc38031e38b381116d1

**Pull request merged**